app/handlers/sessions_create.rb

```diff
@@ -80,8 +80,8 @@ def handle
         authentication_user = users_matching_oauth_data.first
         status = :transferred_authentication
       else
-        outcome = run(CreateUserFromOmniauthData, @data)
-        authentication_user = outcome.outputs[:user]
+        # This will be updated to use a routine that does more
```

Comment:
I think this entire `else` block is probably going to be pulled out in a subsequent, since we will no longer have a way to create an account by a first-time log in with social